containers/datalab/content/run.sh

```diff
@@ -29,5 +29,14 @@ fi
 # Setup environment variables.
 . /datalab/setup-env.sh
 
+# Run the user's custom extension script if it exists. To avoid platform issues with 
+# execution permissions, line endings, etc, we create a local sanitized copy.
+if [ -f /content/datalab/startup.sh ]
```

Comment:
startup.sh may show up in the home notebook list? Can we make it hidden?

Perhaps it is time to move the home dir to /content/datalab/home or /content/datalab/content?